analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/ConditionEvaluatesToConstantBase.cs

```diff
@@ -26,9 +26,11 @@ public abstract class ConditionEvaluatesToConstantBase : SymbolicRuleCheck
 {
     protected const string DiagnosticId2583 = "S2583"; // Bug
     protected const string DiagnosticId2589 = "S2589"; // Code smell
-    protected const string MessageFormat = "Change this condition so that it does not always evaluate to constant value.";
-    protected const string MessageFormatBool = "Change this condition so that it does not always evaluate to '{0}'.";
-    protected const string MessageNull = "Change this expression which always evaluates to 'null'.";
+
+    protected const string MessageFormat = "{0}";
+    private const string MessageBool = "Change this condition so that it does not always evaluate to {1}";
+    private const string MessageNullCoalescing = "Change this expression which always evaluates to the same result.";
```

Comment:
Add tests for this message as well.

Reply:
There are tests for this already - not sure which case I've missed. :/ 
See [TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs](https://github.com/SonarSource/sonar-dotnet/pull/7698/files#diff-e7eedcb54c12bdb191c8a22ea0108ee1c01d7665a6a3d8a8a4b83950d7f6c23e)
below lines 2675-2680.

Reply:
They do not test for the message, though. Or am I overlooking something?